docs/tutorials/summer_triangle.rst

```diff
@@ -169,7 +170,7 @@ window:
     '2015-06-16 15:47:36.466'
 
 So, our targets will be visible (as we've defined it above) from
-'2015-06-15 20:23:40.991' to '2015-06-16 05:47:36.466' US/Hawaii.  Depending on
+2015-06-15 20:23:40.991 to 2015-06-16 05:47:36.466 Hawaii time.  Depending on
 our observation goals, this window of time may be good enough for preliminary
 planning, or we may want to optimize our observational conditions.  If the
 latter is the case, go on to Optimal Observation Time.
```

Comment:
This should probably link to the section or say `...go on to Optimal Observation Time (below).`